ctru-rs/examples/movement.rs

```diff
@@ -34,13 +36,17 @@ fn main() {
         // Be careful: reading without activating the sensors (as done before this loop) will result in a panic.
         println!(
             "\x1b[3;0HAcceleration: {:?}              ",
-            hid.accelerometer_vector()
-                .expect("could not retrieve acceleration vector")
+            Into::<(i16, i16, i16)>::into(
```

Comment:
Can this be `From::from` ?

```suggestion
            <(i16, i16, i16)>::from(
```

Reply:
Yeah it should be, sorry for the inconvenience.